<launch>
    <!-- ARGS -->
    <arg name="ros_pkg" default="aims_als" />

    <node pkg="$(arg ros_pkg)" name="decision_maker_node" type="decision_maker_exe" output="screen">
        <param name="Hz_of_FSM"     value="50.0"/>
        <param name="Hz_of_MONITOR" value="1.0"/>
        <param name="sitl"          value="false"/>

        <param name="epsilon_dt"          value="0.5"/>
        <param name="epsilon_roll"       value="7.5f"/>
        <param name="epsilon_pan"       value="30f"/>
        <param name="epsilon_range"       value="3.5"/>
        <param name="epsilon_speed"       value="4.0"/>
        <param name="epsilon_swing_cnt"   value="2"/>
        <param name="epsilon_swing_angle" value="40."/>

        <param name="vf_phi_des"    value="-27.5"/>
        <param name="vf_phi_delta"  value="34.375"/>

        <param name="landing_condition_gv_speed" value="5.0"/>
        <param name="landing_condition_duration" value="4.0"/>

        <param name="landed_condition_critical_altitude" value="0.30"/>
    </node>
</launch>
